apps/zk-explorer/constants/index.ts

```diff
@@ -4,5 +4,11 @@ export const GITHUB_LANGUAGE_COLORS_API_URL =
 export const API_PREFIX = '/api';
 export const ITEMS_PER_PAGE = 12;
 
+export const FEEDBACK_URL =
+  'https://github.com/webb-tools/webb-dapp/issues/new/choose';
```

Comment:
We already have this constant in the `libs/webb-ui-components/src/constants/index.ts:77`.